CHANGELOG.mkd

```diff
@@ -3,10 +3,23 @@ CHANGELOG
 
 Unreleased
 ----
-- Add exec environment source type. The exec source type allows for the implementation of external environment sources [#1042](https://github.com/puppetlabs/r10k/pull/1042)
+
+
+3.5.0
+-----
+
+- Add exec environment source type. The exec source type allows for the
+  implementation of external environment sources
+  [#1042](https://github.com/puppetlabs/r10k/pull/1042).
+- Improve atomicity of .r10k-deploy.json writes. Fixes
```

Comment:
If we're linking to PRs, let's be consistent. 😊 https://github.com/puppetlabs/r10k/pull/1039

Reply:
tbh, I only linked to the one PR because its impossible to tell what's going on from the ticket itself. I don't think linking folks to code is generally a best practice (its got a little bit of a if you want to know go rtfm vibe), but I didn't want to re-write someone JIRA ticket rn.

Reply:
If your preference is to remove the note about the one that includes a PR in it, then I think that would get us back to consistency  ( :

Reply:
tbh I agree with you that linking to code in the changelog feels kinda silly.. Just trying to understand what our standards are. In that case, I don't think we should link to the other ones..